pkg/tagger/tagger.go

```diff
@@ -31,6 +31,8 @@ type Tagger struct {
 
 // NewTagger returns an allocated tagger. You still have to run Init()
 // once the config package is ready.
+// This is exported for system tests, you are probably looking for
+// tagger.Tag() using the global instance instead of creating your own.
 func NewTagger() (*Tagger, error) {
```

Comment:
I might be wrong but I think you can even remove exportation of this one, you'll still be able to call it from your tests because you're in the same package

Reply:
We don't yet have integration/system tests on that package, you're right. We'll export if later if required.